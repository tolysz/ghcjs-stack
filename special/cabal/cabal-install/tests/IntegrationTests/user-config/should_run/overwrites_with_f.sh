. ../common.sh

rm -f ./cabal-config
cabal --config-file=./cabal-config user-config init \
    || die "Couldn't create config file"
cabal --config-file=./cabal-config user-config -f init \
    || die "Couldn't create config file"
test -e ./cabal-config || die "Config file doesn't exist"
rm -f ./cabal-config
