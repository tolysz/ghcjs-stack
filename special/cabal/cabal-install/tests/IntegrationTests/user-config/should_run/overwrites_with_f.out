Writing default configuration to ./cabal-config
Writing default configuration to ./cabal-config
